### ISSUE TITLE
Error when exporting with minify true

### ISSUE THREAD

**Issue by im-sticky** [sofn-xyz/mailing#206]
First off I would like to say how much I appreciate the fast turnaround on the export minify option. However I seem to be running into an error when it is enabled, both on my project as well as the examples that get created when running `mailing` for the first time.

The error is as follows:
```
mailing Exporting minified preview html to
mailing ./previews_html
mailing   |-- text_email_reset_password.html
mailing   |-- welcome_to_amelita.html
mailing ✅ Processed 2 previews
/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:1928
                          if (errors.length) {
                                     ^

TypeError: Cannot read properties of undefined (reading 'length')
    at /dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:1928:38
    at Array.map (<anonymous>)
    at _callee$ (/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:1919:78)
    at tryCatch (/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:141:17)
    at Generator._invoke (/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:121:24)
    at Generator.next (/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:172:21)
    at asyncGeneratorStep (/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:405:24)
    at _next (/dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:427:9)
    at /dev-testing/mailing-demo/node_modules/mailing/dist/mailing.cjs.dev.js:434:7
    at new Promise (<anonymous>)
```

Despite saying at least those 2 of the 4 previews are processed there is not HTML output.

package.json
```
{
  "dependencies": {
    "mailing": "^0.7.9",
    "mailing-core": "^0.7.9",
    "next": "^12.3.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  }
}
```

mailing.config.json
```
{
  "typescript": false,
  "emailsDir": "./emails",
  "outDir": "./previews_html",
  "anonymousId": null,
  "minify": true
}
```

Code aside from minify option is not altered in any way after running initial `mailing` command.

**Comment by alexfarrill** [comment]
@im-sticky Thanks for getting in touch about this.

We actually had to refactor the minify code for our next release because mjml-core has deprecated the minify option in future versions.  Minification will still work in the same way in exportPreviews, just a slightly different thing happens under the hood, but it will use the same minification library that mjml-core uses in the current version.

I wonder if you would try these beta releases and see if either of them fix the problem:
mailing@0.7.10-next.0 (refactor the minification as described above)
mailing@0.7.10-next.1 (I also added a ? to the .length in case the type the mjml library declares is somehow incorrect)

Thanks for using mailing!
